charts/kubefed/charts/controllermanager/templates/deployments.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     kubefed-control-plane: controller-manager
 {{- if .Values.controller.annotations }}
   annotations:
+    "helm.sh/hook": pre-install, pre-upgrade
```

Comment:
Marking the deployment as a hook doesn't feel right to me. This will not be cleaned up on helm uninstall.

Reply:
I just ran helm uninstall, and the resources were all cleaned up 🤔.

Reply:
Okay, I see what is happening.